test/utils.py

```diff
@@ -188,6 +188,29 @@ def gnoi_reboot(method, delay, message):
     ret, msg = run_cmd(cmd)
     return ret, msg
 
+def gnoi_kill_process():
+    path = os.getcwd()
+    json_data = '{"name": "snmp"}' 
+    cmd = path + '/build/bin/gnoi_client '
+    cmd += '-insecure -target 127.0.0.1:8080 '
+    cmd += '-rpc KillProcess '
+    cmd += f'-jsonin \'{json_data}\''
+    ret, msg = run_cmd(cmd)
+    return ret, msg
+
+def gnoi_restart_process(valid):
+    if not valid:
+        json_data = '{"name": "snmp", "restart": invalid}'
```

Comment:
Does invalid exist in boolean? or you may want ot use string instead.

Reply:
invalid does not exist in boolean, but this is meant to test a failure scenario with invalid option passed. For example, this is the expected outcome when running gnoi_client with an invalid option passed:

```
root@str-s6100-acs-1:/# gnoi_client -target 127.0.0.1:50052 -logtostderr -insecure -rpc KillProcess -jsonin '{"name": "snmp", "restart": invalid}'    
Kill Process with optional restart
panic: invalid character 'i' looking for beginning of value

goroutine 1 [running]:
main.killProcess(0xb4cbc0, 0xc00027e150, 0xb47980, 0xc00007f240)
        /sonic/src/sonic-gnmi/gnoi_client/gnoi_client.go:120 +0x2b4
main.main()
        /sonic/src/sonic-gnmi/gnoi_client/gnoi_client.go:61 +0x7dd
```